webapp/components/topbar/dialogs/LaunchDialog.js

```diff
@@ -241,16 +242,23 @@ const LaunchDialog = (props) => {
   const wait = (milliseconds) => new Promise((resolve) => setTimeout(resolve, milliseconds));
   const handleConfigurationUpdate = (e) => {
     e.stopPropagation();
-
     setLoading(true);
-
-    // TODO: @Muhaddatha save runConfig in backend
-    wait(2000).then(() => {
+    ExperimentsApi.sendExperimentConfiguration(runConfig).then(() => {
       setLoading(false);
       setExpandConfiguration(false);
+    }).catch((error) => {
+      // handle error
```

Comment:
for now you can just use `setLoading(false)` and log the error to the console, we show a popup with an error message in a later stage